main.js

```diff
@@ -1,3 +1,4 @@
+// portfolio-mobile menu
```

Comment:
Also, kindly fix the typo here:

<img width="317" alt="image" src="https://user-images.githubusercontent.com/34578631/162697744-d94941fc-7586-41db-b1d1-1d0d0ea9bed5.png">